<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgProducts</class>
 <widget class="QWidget" name="wdgProducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>516</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Products list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Búsqueda por código, ISIN o nombre del producto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt"/>
       </item>
       <item>
        <widget class="QToolButton" name="cmd">
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/search.png</normaloff>:/xulpymoney/search.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="myQTableWidget" name="tblInversiones">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Code</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Investment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ISIN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last update</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>% Diario</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>% Inicio año</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>% Dividendo</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFound">
       <property name="text">
        <string>Registers found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionProductReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/history.png</normaloff>:/xulpymoney/history.png</iconset>
   </property>
   <property name="text">
    <string>Product report</string>
   </property>
   <property name="toolTip">
    <string>Product report</string>
   </property>
  </action>
  <action name="actionOrdenarTPCDiario">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Diario</string>
   </property>
  </action>
  <action name="actionOrdenarTPCAnual">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Anual</string>
   </property>
  </action>
  <action name="actionOrdenarName">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </action>
  <action name="actionOrdenarDividendo">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Dividendo</string>
   </property>
  </action>
  <action name="actionOrdenarHora">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Hora</string>
   </property>
  </action>
  <action name="actionIbex35">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/countries/spain.gif</normaloff>:/countries/spain.gif</iconset>
   </property>
   <property name="text">
    <string>Ibex 35</string>
   </property>
  </action>
  <action name="actionProductNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New product</string>
   </property>
  </action>
  <action name="actionProductEdit">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/document-edit.png</normaloff>:/xulpymoney/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit product</string>
   </property>
  </action>
  <action name="actionProductDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete product</string>
   </property>
   <property name="toolTip">
    <string>Delete product</string>
   </property>
  </action>
  <action name="actionFavoritos">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/star.png</normaloff>:/xulpymoney/star.png</iconset>
   </property>
   <property name="text">
    <string>Añadir a favoritos</string>
   </property>
   <property name="toolTip">
    <string>Añadir a favoritos</string>
   </property>
  </action>
  <action name="actionMergeCodes">
   <property name="icon">
    <iconset>
     <normaloff>:/images/cakes.png</normaloff>:/images/cakes.png</iconset>
   </property>
   <property name="text">
    <string>Combina códigos seleccionados</string>
   </property>
  </action>
  <action name="actionQuoteNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Nueva cotización</string>
   </property>
   <property name="toolTip">
    <string>Nueva cotización</string>
   </property>
  </action>
  <action name="actionEstimationDPSNew">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>New DPS estimation</string>
   </property>
   <property name="toolTip">
    <string>New Dividend per share estimation</string>
   </property>
  </action>
  <action name="actionPurge">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
   </property>
   <property name="text">
    <string>Purge investment</string>
   </property>
   <property name="toolTip">
    <string>Deletes quotes innecesary. Leaves open, high, low and close quotes.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txt</tabstop>
  <tabstop>tblInversiones</tabstop>
 </tabstops>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
